app/Main.hs

```diff
@@ -412,23 +412,44 @@ checkForUpdates applyMinorUpdates applyMajorUpdates = do
     isMinorReleaseFrom (x : xs) (y : ys) = y == x && ys > xs
     isMinorReleaseFrom _        _        = False
 
+verify :: Maybe String -> IO ()
+verify Nothing = echoT "A package name is required for verify"
```

Comment:
If it's required, why not remove the `Maybe`?

Reply:
ohhh, does it work that way? oops...